otherlibs/site/src/plugins/linker/dune

```diff
@@ -0,0 +1,5 @@
+(library
+ (name linker)
```

Comment:
I will change the name of the library to something prefixed with `dune_site` to prevent name clashes.